src/models/aero_state.jl

```diff
@@ -75,8 +75,10 @@ function AeroState(tas::Number, alpha::Number, beta::Number, height::Number)
     AeroState(alpha, beta, 0, tas, eas, cas, ias, qinf, mach)
 end
 
-
-function state_aerostate(pos, att, tas, alpha, beta, env=DefaultEnvironment(),
+"""
+gives both states and aerostates from the inputs provided
+"""
+function state_aerostate(pos, att, tas, alpha, beta, env::Environment,   # =DefaultEnvironment(), # not defind
```

Comment:
Good point with `DefaultEnvironment` I need a refactor tool, doing it manually has introduced many bugs... Thanks!

Reply:
so is it better to delete this instead of commenting out ?